test_cli.py

```diff
@@ -154,7 +154,12 @@ def wait_for_contents(uuid, substring, timeout_seconds=100):
 
 
 def wait(uuid, expected_exit_code=0):
-    _run_command([cl, 'wait', uuid], expected_exit_code)
+    try:
+        _run_command([cl, 'wait', uuid], expected_exit_code)
+    except AssertionError as e:
+        _run_command([cl, 'info', uuid])
```

Comment:
Why are we adding this?  What are the errors and is there a better way to handle them?

Reply:
It's much easier to debug when the bundle fails on CI, because running `cl info` will then show the reason why the bundle has failed. The error is caused when the exit code is not equal to the expected exit code.

This change still causes the same error to be raised, but we just get the results of `cl info` printed to CI before that happens.

Reply:
What do you think of instead passing in a flag to `_run_command` that says if there's a failure to print out information about the bundle?  I'm guessing this type of output might be useful in many cases, not just here.

Reply:
Interesting idea -- how would we get the bundle uuid though? I think just assuming that the third argument is equal to the bundle uuid may end up not being a good heuristic?

Reply:
A uuid can be passed in as an optional argument.

Reply:
Let's do this in a separate PR.